docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
 executions.
 
 
+.. _`Defining the WBEM Server`:
+
+Defining the WBEM Server
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The target WBEM server  can be defined on the command line in several ways with
+the in the following arguments :
+
+1. Define the WBEM server characteristics with a set of general options
+   The geneal options that define the WBEM Server include the:
+
+   * The :ref:`--server general option` that defines the URI of the WBEM server
+   * The :ref:`--default-namespace general option`, that defines the nameespace
+     used if no namespace is included with each command,
+   * several connection security general options:
+
+     * The :ref:`--user general option`,  defines the WBEM server user name
+     * The :ref:`--password general option`, defining the password for the WBEM
+       server user
+     * The :ref:`--no-verify general option` that defines whether the client verifies
+       certificates received from the WBEM server,
+     * The :ref:`--certfile general option`, client certificate file,
+     * The :ref:`--keyfile general option`,  client key file,
+     * The :ref:`--ca-certs general option`), a collection of certificates against
+       which certificates received from the WBEM server are verified.
+
+   * The :ref:`--timeout general option` that defines the client side connection
+     timeout
```

Comment:
It is ot the connection timeout, but the operation timeout.

Reply:
TKS. Done